### ISSUE TITLE
Incorrect tap area for right-to-left languages

### ISSUE THREAD

**Issue by William-Coates** [mapbox/mapbox-search-ios#184]
When a device is using a right-to-left language, in order to switch between 'Favorites' and 'Categories', you have to tap on the currently selected tab in order to switch tabs.

Also, is there a way for to localize the strings in Mapbox search, or does that need to be done by Mapbox?

![SCR-20240228-mmqs](https://github.com/mapbox/mapbox-search-ios/assets/149601233/a81b51e8-f79f-403c-889e-9cd825fbb14f)

**Comment by aokj4ck** [comment]
@William-Coates thank you for this bug report! I've opened a pull request to add Right-to-Left language support to this component.

1. Please feel free to change your MapboxSearch package import to `branch = 184-incorrect-tap-area-for-right-to-left-languages` to evaluate this fix and report any feedback. 

2. I anticipate that this fix will be included in our next 2.0.0-rc.3 release which is in-progress at PR https://github.com/mapbox/mapbox-search-ios/pull/190.

> Also, is there a way for to localize the strings in Mapbox search, or does that need to be done by Mapbox?

3. I need to do some more investigation and report back to answer this question.

**Comment by aokj4ck** [comment]
@William-Coates please let me know if that merged PR into `main` solves the problem you were seeing. I am working on getting an answer for # 3 about localized strings.

**Comment by William-Coates** [comment]
@aokj4ck Yes, this now works, thank you! Although it's a shame I won't be able to use it yet, stuck on 1.0.0 due to Mapbox Navigation. I suppose I could fork this repo and add the fix to 1.0.0 myself?

**Comment by aokj4ck** [comment]
@William-Coates ah, yes, my apologies. I agree that a fork would probably be the most expedient path. You may have an easier time applying the files through git, excluding .xcodeproj changes, and then adding those files to the .xcodeproj build manually. Let me know if you have any fork blockers in a new issue.

In the past we had suggested a fork to add localization strings (see https://github.com/mapbox/mapbox-search-ios/issues/40#issuecomment-1089349575) and I am seeking a more complete / easier solution. Right now I do not have news to share yet. I'll keep this issue open until I can get an answer.

At this time we have been focusing on the 2.0.0 series. If you have additional needs for the 1.0.0 release series please open additional issues (and reach out to your account representative if applicable). For example if you must have Xcode compatibility updates in the 1.0.0 series then having an open issue would help greatly.

**Comment by aokj4ck** [comment]
@RL-William-Coates this fix should be included in [the latest v1 release which is 1.0.0-rc.9](https://github.com/mapbox/mapbox-search-ios/releases/tag/v1.0.0-rc.9). Please try out that version and let me know if this solves the ticket.
